### ISSUE TITLE
Clarifications about IP Ban

### ISSUE THREAD

**Issue by Aerial87** [finndev/PokeBuddy#728]
Hi all, 
it semms today Niantic starts a Banhammer on IP adress that was running multiple accounts (not on account themselves).
In this post i will report what i actually get, but it would be grat if you could also help me to understand some mechanics.

I was running 3 bot at the same time (2 pokebuddy, and 1 other bot). Today all of them was unable to find anything and just roam around meaningless.
I switched provider (i got 2 different internet connections at home) and now it works fine, so accounts are not banned, just IP.
In this forum i read that running 2 or more Pokebuddy may cause IP ban, what i'm asking is:
-if i run just 1 pokebuddy and 1 other bot may i still get banned?
-if so, nox app (android device emulator) still count as a multiple device for ip ban?
-how much ip ban stays long?

Thanks all for your help, i hope this post will collect as many usefull contribution as possibile.

**Comment by Thorben24** [comment]
-if i run just 1 pokebuddy and 1 other bot may i still get banned?
Yes I think so.
-if so, nox app (android device emulator) still count as a multiple device for ip ban?
Yes I think so.
-how much ip ban stays long?
1-6 hours.

**Comment by mcmagic1** [comment]
@Aerial87 how can you login in poke buddy ? i write FREE-BETA but it's not working. it say's LOGIN FAILED. do you have a solution here ?

**Comment by MudlaffP** [comment]
I played with one account and get ban on IP. Stopped for something like 2-3 h then try again (this time I unselected incubators)  and after 30 min get next ban on ip. Now I stop Poke Buddy until some solution will be found.

About login i did not have problem with FREE-BETA (I updated Poke Buddy after each ip ban).

**Comment by Aerial87** [comment]
ATM i'm running 1 pokebuddy + 1 other bot for 26 minutes, it seems work fine. I'll update if i get banned.

**Comment by Aerial87** [comment]
@mcmagic1 No login problems here, did u try to re-download the client?

**Comment by matt2310** [comment]
i couldnt find any pokestops or pokemon around 7 AM est, using one pokebuddy ap and switching between two accounts....
how should i run it after work so i dont get banned?

**Comment by ukyo28** [comment]
No problems here either. been running 2hrs already. 1 pokebuddy running only.

![capture](https://cloud.githubusercontent.com/assets/20924230/17553311/e344d666-5f0d-11e6-8f56-402c54474ef8.JPG)

**Comment by Aerial87** [comment]
@matt2310 I Guess your IP got banned.
Now you can switch IP of just wait (1-6 hours) to be unbanned.

**Comment by matt2310** [comment]
im at work and have a cat scan today so i can wait the time, but i wanna know how to run and not get banned... im new to all this i dont even really understand VPN and how to use that

**Comment by Aerial87** [comment]
Here is a guide for VPN, but i did not tryed it yet.
http://www.ownedcore.com/forums/pokemon-go/pokemon-go-hacks-cheats/570331-quick-ip-unban-vpn-ip-no-ban.html

**Comment by MariaBitterlich** [comment]
I started PokeBuddy on PC with an new bot-google account and it was running fine. I wanted to send in the Pokémon with IP < 80% and therefore:
1. Switched from normal-google account to bot-google account on iPhone
2. Deleted a couple of Pokémon with iPhone on bot-google account manually

Around this time I might be soft banned on **both** google accounts: I couldn't see any arena or pokéstop for a while.

Was this a softban?
How could the normal-google account could be softbanned? Just by logging out/in with both google-accounts from the same iPhone?

**Comment by Aerial87** [comment]
@MariaBitterlich Did both account was ever connected at the same time?

UPDATE ABOUT BOTTING: After 1:26 hours i get IP banned using Pokebuddy and the other bot together.
Next try will be using just 1 pokebuddy on a fresh ip.

**Comment by MariaBitterlich** [comment]
@Aerial87, yes normally I run my normal account on iPhone for playing and started today with bot-account on PC. There running again in parallel, but on different devices.

**Comment by Aerial87** [comment]
@MariaBitterlich If both are connected trought the same provider i guess your IP can be banned, doesn't matter if playng on different devices.
I had to wait about 1 hour to get my 1st IP unbanned.

**Comment by MariaBitterlich** [comment]
@Aerial87, yes, I'm banned on both devices again - there were connected via same wifi.

**Comment by Thorben24** [comment]
The VPN guide should work btw, cant you change your ip adress by yourself, or is it not dynamic?

**Comment by MariaBitterlich** [comment]
@Aerial87, I changed my iPhone from wifi to LTE and the normal account is working again without waiting time. The PC bot is still banned, so I'm sure that's an IP ban.

I don't want to risk permanent bans on my normal account, so I better play with iPhone and normal account not via the same wifi connection as the bot, right?

**Comment by Aerial87** [comment]
I think it's better play as safe as possible now, i'm just using smurf accounts to investigate. I hope someone will find the pattern for this IP ban issue.

**Comment by Thorben24** [comment]
Often it can be fixed with botting only in the country/state your ip adress is located. But not 100% sure about that, just try it.

**Comment by Aerial87** [comment]
1 pokebuddy running for 1:30 h, still working without VPN in another country.

**Comment by Thorben24** [comment]
Thats great, im running 3 pokebuddy accounts right now in sydney, new york and tokyo since around 3 hours. Got 1 german and 2 different french ip adresses. Everything working fine for me, too.

**Comment by MariaBitterlich** [comment]
I'm seperated my IPs: normal account LTE, bot account WIFI. I guess my problems started as I was using my iPhone and PC via the same WIFI at the same time, but now I've started them both again with different IPs.

**Comment by Shut-in-room** [comment]
let me remind you about your walking speed. i ran the PB on 20- speed for like 9-10 hours yesterday was all good..

i notice on some settings of PB... people use like 20++ walking speed which is not normal for a person to just like zoom around on the map.. my guess is that you should be at 20 below speed setting up 20+ or more.. PB would be a ban hazard = suggestion.. try to get a Virtual PC installed on you laptop and use that one to run the 2nd PB..havent tried this but will do later.. remember that the logarithmic thinking of the AI on Pokemon Go is as simple as.. hey look someone is running around with a speed of .. wait wtf! 50km/hr non stop = thats not a normal person might be an alien on flying shoes.. ok lets soft ban them..  that is why they updated on that pop-up that says you are going to fast.. my guess is that if you dont click that or you did click that you go on a watch list of using bots.. so do not go over 20 that would be a stupid thing to do lol
last piece of advice AVOID THE DANG POP-UP NOTICE!
